test/functional/test.sh

```diff
@@ -21,6 +21,34 @@ do
     fi
 done
 
+declare -a sites_convert=("test_site_convert")
+
+for site in "${sites_convert[@]}"
+do
+    # print site name
+    echo
+    echo "Running $site tests"
+
+    # convert site
+    node ../../index.js init "$site" -c
+
+    # build site
+    node ../../index.js build "$site"
+
+    # run our test script to compare html files
+    node testUtil/test.js "$site" "test_site_convert_expected"
+
+    # delete generated files
+    rm -rf "$site"/_markbind "$site"/_site
+    rm "$site"/about.md "$site"/index.md "$site"/site.json
+
+    if [ $? -ne 0 ]
```

Comment:
Same problem as the Windows' version: the return code `$?` belong to the `rm` command, rather than the `node testUtil/test.js` command.